src/layouts/RootLayout/RootLayout.tsx

```diff
@@ -1,17 +1,17 @@
-'use client';
+"use client";
 
-import React from 'react';
-import { CssBaseline } from '@mui/material';
-import '@fontsource/roboto/300.css';
-import '@fontsource/roboto/400.css';
-import '@fontsource/roboto/500.css';
-import '@fontsource/roboto/700.css';
-import { createTheme, ThemeProvider } from '../../components/dist';
+import React from "react";
+import { CssBaseline } from "@mui/material";
+import "@fontsource/roboto/300.css";
```

Comment:
We shouldn't need this. Asics fonts should be used instead. Not sure if Colin added them to the Blocks package, but if not, we should add them and use those.